public/typescript/ui/component/App.tsx

```diff
@@ -4,19 +4,23 @@ import { ErrorTable } from './ErrorTable';
 
 export interface AppProps {
   t: (key: string) => string; requestInformation: RequestInformation;
+  started: boolean;
+  startMessage: TranslationString;
   errorInfo: ErrorInformation | null;
   message: TranslationString;
   onClick: () => void;
+  onStart: () => void;
 }
 
-export const App: FC<AppProps> = ({ t, message, requestInformation, errorInfo, onClick }) => {
+export const App: FC<AppProps> = ({ t, started, startMessage, message, requestInformation, errorInfo, onClick, onStart }) => {
   return (
     <div>
       <p>
         {t(message)}
       </p>
-      {errorInfo && <ErrorTable t={t} errorInfo={errorInfo} clientInfo={requestInformation} />}
-      {errorInfo && errorInfo.showRetry && <button onClick={onClick}>{t('retry')}</button>}
+      {started && errorInfo && <ErrorTable t={t} errorInfo={errorInfo} clientInfo={requestInformation} />}
+      {started && errorInfo && errorInfo.showRetry && <button className="btn btn-primary" onClick={onClick}>{t('retry')}</button>}
+      {!started && <button className="btn btn-primary" onClick={onStart}>{t(startMessage)}</button>}
```

Comment:
I don't see any announcement for screen-reader users, or a focus change for keyboard users . So likely they won't know what's happening?  Then again, i reckon stepup has not been accessibility-proofed yet, so likely a separate effort is needed to make it accessible.